src/config/texts.js

```diff
@@ -1156,6 +1156,7 @@ export const texts = {
     lunch: 'Gastro',
     surveys: 'Umfragen',
     water: 'Wasser',
-    weather: 'Wetter'
+    weather: 'Wetter',
+    web: 'Web'
```

Comment:
this should be an empty string or removed entirely. it makes no sense to write "Web" for a widget.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/932/commits/3f34b61ffadcc901aa1b099ea50b50fff3ff51fb